{"uuid":"c940589a-bcc9-4c4d-8c08-b55f7389ea41","historyId":"98b4060c5c5e602554b087f592293d8d","fullName":"com.cyberiansoft.test.bo.testcases.BackOfficeOperationsServiceRequestsTestCases.testMiscellaneousEventsAppointmentFailed","labels":[{"name":"package","value":"com.cyberiansoft.test.bo.testcases.BackOfficeOperationsServiceRequestsTestCases"},{"name":"testClass","value":"com.cyberiansoft.test.bo.testcases.BackOfficeOperationsServiceRequestsTestCases"},{"name":"testMethod","value":"testMiscellaneousEventsAppointmentFailed"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"TestAutomationProject"},{"name":"subSuite","value":"com.cyberiansoft.test.bo.testcases.BackOfficeOperationsServiceRequestsTestCases"},{"name":"host","value":"DESKTOP-RNHDCNI"},{"name":"thread","value":"11032@DESKTOP-RNHDCNI.main(1)"}],"links":[],"name":"testMiscellaneousEventsAppointmentFailed","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"failed to connect","trace":"javax.mail.AuthenticationFailedException: failed to connect\r\n\tat javax.mail.Service.connect(Service.java:403)\r\n\tat javax.mail.Service.connect(Service.java:246)\r\n\tat javax.mail.Service.connect(Service.java:267)\r\n\tat com.cyberiansoft.test.email.EmailUtils.<init>(EmailUtils.java:41)\r\n\tat com.cyberiansoft.test.bo.testcases.BackOfficeOperationsServiceRequestsTestCases.testMiscellaneousEventsAppointmentFailed(BackOfficeOperationsServiceRequestsTestCases.java:1231)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)\r\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\r\n"},"stage":"finished","start":1532074123421,"stop":1532074245214,"parameters":[{"name":"arg0","value":"testMiscellaneousEventsAppointmentFailed.01"},{"name":"arg1","value":"Test Case 31296:Miscellaneous - Events: Appointment Failed"},{"name":"arg2","value":"{\"serviceRequestType\":\"Zak_Request_Type\",\"notificationDropDown\":\"ServiceRequest Checked In\",\"emailKeyWord\":\"was created\",\"alert\":\"test appointment Appointment Failed\",\"description\":\"Test Case 31296:Miscellaneous - Events: Appointment Failed\",\"eventNewName\":\"test appointment SR Checked In\",\"technician\":\"Automation 2 Appointment Tech\",\"event\":\"Appointment Failed\",\"serviceRequestGeneralInfo\":\"Automation1 Primary  Tech\",\"selected\":\"test appointment creation\\/fail\",\"rowID\":\"testMiscellaneousEventsAppointmentFailed.01\",\"customer\":\"Automation Wholesale\"}"}]}